docs/how-to-define-transactions.md

```diff
@@ -0,0 +1,18 @@
+# How Transactions Are Defined
+
+Each transaction has its properties defined in `./src/containers/shared/components/Transaction/{TransactionType}/index.ts`.
+
+- **Description**: A React component that defines the "Description" portion of a transaction's "Detailed" tab.
+- **Simple**: A React component that defines the left column of a transaction's "Simple" tab.
+- **TableDetail**: A React component that defines the body of transaction on the account, ledger, token, or nft page.
+- **action**: A TransactionCategory value used to determine shape color of the transaction.
+- **category**: A TransactionCategory value used to determine the color of the transaction.
+- **parser**: A function which takes a transaction object and meta nodes. It returns additional properties to map onto a transaction.
+  This is run when a transaction is received from the server and is useful for transforming complex properties derived from nodes.
```

Comment:
An example here would be quite helpful plus maybe a screenshot of what it looks like in the end?

Reply:
This PR was originally intended to just be about defining how to add new languages.  This was just copied but it looks like git thinks it is a brand new file.  Going to move this to a different PR to update with better documentation later.